plugins/inlinetoolbar/plugin.js

```diff
@@ -80,9 +80,9 @@
 		 * Menu items added to inline toolbar.
 		 *
 		 * @private
-		 * @property {CKEDITOR.ui.button[]/CKEDITOR.ui.richCombo[]}
+		 * @property {Object.<string, CKEDITOR.ui.button|CKEDITOR.ui.richCombo>}
```

Comment:
1. We use backslash as a type separator.
1. We use uppercased `String` when referring to strings.

Reply:
Have you found any traces that this syntax is supported by JSDuck? I can't find that in their docs.

To be honest it's the very first time I see this syntax for annotating dictionaries, but looks okish to me.

I assume you took it from here https://stackoverflow.com/questions/19513955/how-to-document-a-dictionary-in-jsdoc